### PR TITLE
Feat: edit operator

### PR DESCRIPTION
**Pull request by pbastia**

### CODE REVIEW

File: app/lib/pages/relayFormPageFactory.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+) => {
+  function FormPage({
+    preloadedQuery,
+  }: RelayProps<{}, relayFormPageFactoryQuery>) {
+    const { session, formChange } = usePreloadedQuery(
+      pageQuery,
+      preloadedQuery
+    );
+
+    const [updateFormChange, isUpdatingFormChange] = useUpdateFormChange();
```
Comment (matthieu-foucault) [inline app/lib/pages/relayFormPageFactory.tsx:57]: When fixing #285 we might need to make this hook a parameter of the factory

**Review by dleard — CHANGES_REQUESTED** [review]
looks good, just missing a verify

**Review by matthieu-foucault — DISMISSED** [review]

File: app/components/Operator/OperatorForm.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Button } from "@button-inc/bcgov-theme";
+import FormBase from "components/Form/FormBase";
+import operatorSchema from "data/jsonSchemaForm/operatorSchema";
+import { JSONSchema7 } from "json-schema";
+import { FormPageFactoryComponentProps } from "lib/pages/relayFormPageFactory";
+
+const OperatorForm: React.FC<FormPageFactoryComponentProps> = (props) => {
+  return (
+    <FormBase {...props} schema={operatorSchema as JSONSchema7} uiSchema={{}}>
+      <Button type="submit">Submit</Button>
```
Comment (matthieu-foucault) [inline app/components/Operator/OperatorForm.tsx:10]: This button can use a margin on the right

**Review by matthieu-foucault — APPROVED** [review]
@dleard This needs your approval as well

**Review by dleard — APPROVED** [review]

**Pull request merged**